applications/CompressiblePotentialFlowApplication/tests/naca0012_small_compressible_test/cl_ref.dat

```diff
@@ -0,0 +1 @@
+ 0.355163671775
```

Comment:
Same comment.

Reply:
This is solved in https://github.com/KratosMultiphysics/Kratos/pull/4668